### ISSUE TITLE
Redux 4 is out

### ISSUE THREAD

**Issue by mandx** [4Catalyzer/farce#59]
[Redux 4 is out](https://github.com/reactjs/redux/releases/tag/v4.0.0), but when using Found/Farce with a project using Redux@4, the [duplicate-package-checker-webpack-plugin](https://www.npmjs.com/package/duplicate-package-checker-webpack-plugin) warns me about my app bundling Redux@4 and Farce bundling Redux@3.7.

As a workaround (for me) I'm using Webpack's `resolve.alias` setting, to force bundling just one version of Redux, and everything seems to work fine:
```
...
resolve: {
  alias: {
    redux: path.resolve(__dirname, 'node_modules/redux'),
  },
},
...
```
I guess I just want a ticket to subscribe to, so I know when to upgrade to the new Found/Farce versions and then remove the Webpack workaround.

**Comment by taion** [comment]
Cool! I'll take a look at bumping the dep when I get a chance.

**Comment by taion** [comment]
Released in v0.2.6.
